conf/modules.config

```diff
@@ -412,4 +412,12 @@ process {
             pattern: '*.flagstat'
         ]
     }
+
+    withName: MTNUCRATIO {
```

Comment:
I don't think we need to publish this, it's only useful in the final MultiQC report (so just need to make sure that is included in the MultiQC config later :) )

Reply:
Yup that's fine :)